### DISCUSSION TITLE
How move nodes from "left" to "right" side of the map?

### DISCUSSION THREAD

**Discussion by nolsum** [freeplane/freeplane#1186] [Help]
Hi!

This feels like a stupid questions, but I actually can not find out how to do it. I have a map where for some reason ALL nodes are on the left side of the core node, see here: 

![Skarmklipp](https://github.com/freeplane/freeplane/assets/86204306/b20f5bb1-a4cd-499d-8545-974f1a54fa4d)

How do I move some nodes to the right side - so their child nodes also unfold to the right? I have tried to drag a node to the right side of the central node, but it does not help. Is there a setting or something for this?

Magnus

**Comment by sifran-github** [comment]
> I have tried to drag a node to the right side of the central node

this should be the answer.

if you are using the new version of freeplane, check the layout of the root node and make sure it is configured to allow nodes on both sides.

**Comment by nolsum** [comment]
Hi!

Thanks!

I'm using version 1.7.10 on Ubintu. Where do I find the settings for the layout of the root node? I can not find anything that allows me to configure which side it would allow nodes.... so where would I find it?

Magnus

**Reply by sifran-github** [reply]
you are using an old version that doesn't support layouts, so it should work.

you should be able to drag and drop a node to here (right side of the root node) to make it go right:
![image](https://github.com/freeplane/freeplane/assets/5730355/9a5ce802-4dbc-4451-8708-c13086168259)

you can also download a new version if you want.

**Comment by EdoFro** [comment]
@nolsum,
you can also move a node using Ctrl + arrow keys.

**Comment by nolsum** [comment]
Hi!

Thanks. THe Ctrl + arrow solved it! 

Magnus

**Comment by LutzFassl** [comment]
I noticed a change of behavior in the newer version that I installed.
_This is concerning the Mouse X-Position:_
Previously to move a node up/down I could drag and drop it and with the mouse - I was able to move e.g. over the front or the middle area  of the target node to drop it above it.

**But now** with the newer versions I have to drop it over the MIDDLE section of the target node. This can be irritating for long nodes. Can I activate the previous behavior where also dropping it over the front of the label was allowed?
